boot/bootutil/src/image_validate.c

```diff
@@ -182,7 +186,7 @@ bootutil_img_validate(struct image_header *hdr, const struct flash_area *fap,
     int key_id = -1;
 #endif
     struct image_tlv tlv;
-    uint8_t buf[256];
+    uint8_t buf[384];  /* XXX RSA3072 has the largest key! */
```

Comment:
shouldn't the size be compile time optional? RSA3072 might be not enabled.

Reply:
Could be, it was previously sized to the largest possible key; could also be smaller than 256 for EC.

Reply:
This should be OK now.